publisher-sdk/src/main/java/com/criteo/publisher/model/CdbResponseSlot.kt

```diff
@@ -0,0 +1,83 @@
+/*
+ *    Copyright 2020 Criteo
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.criteo.publisher.model
+
+import com.criteo.publisher.Clock
+import com.criteo.publisher.DependencyProvider
+import com.criteo.publisher.annotation.OpenForTesting
+import com.criteo.publisher.model.nativeads.NativeAssets
+import com.criteo.publisher.util.URLUtil
+import com.google.gson.annotations.SerializedName
+import org.json.JSONObject
+import java.io.ByteArrayInputStream
+
+@OpenForTesting
+data class CdbResponseSlot(
```

Comment:
Why was `Slot` renamed to `CdbResponseSlot` during the data class migration? Is there another `Slot` class?
I prefer `Slot` for simplicity.

Reply:
The is the `CdbRequestSlot` too. This is to make the distinction.